### PR TITLE
Use nova service-list for newer Cloud versions

### PR DESCRIPTION
**Pull request by dirkmueller**
The old nova-manage command is deprecated

### CODE REVIEW

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -3478,10 +3478,14 @@ function oncontroller_testsetup()
...
     # wait for nova-manage to be successful
-    for n in $(seq 1 200) ;  do
-        test "$(nova-manage service list  | fgrep -cv -- \:\-\))" -lt 2 && break
-        sleep 1
-    done
+    if iscloudver 7plus; then
+        wait_for 200 1 "test \"$(nova service list  | fgrep -cv -- \ up\ )\" -lt 5" "Nova services to be up and running"
+    else
+        for n in $(seq 1 200) ;  do
+            test "$(nova-manage service list  | fgrep -cv -- \:\-\))" -lt 2 && break
```
Comment (jsuchome) [inline scripts/qa_crowbarsetup.sh]: Could you use `wait_for` here as well?

Comment (dirkmueller) [inline scripts/qa_crowbarsetup.sh]: sure, just didn't want to break the old case (or rather, spend the testing time on it)

**Comment by toabctl** [comment]
+1

**Comment by jsuchome** [comment]
+1 (not tested)

**Pull request merged**